CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+Version 340:
+
+* buffers have a features table in the docs
+* buffer_ref allows using beast buffers with asio
+* buffer_adaptor works with multi-buffer
```

Comment:
just squash the changelog commit into the last commit of the PR we don't need to see this clogging up the commit log